add more context in assigning licenses for auth tokens (#1000)
localstack/docs: content/en/getting-started/auth-token/index.md

For more information about starting LocalStack with Docker, take a look at our [Docker installation](https://docs.localstack.cloud/getting-started/installation/#docker) guide.

### Docker Compose

> To start LocalStack using `docker-compose`, you have to include the `LOCALSTACK_AUTH_TOKEN` environment variable in your `docker-compose.yml` file:

Reviewer: ```suggestion
To start LocalStack using `docker compose`, you have to include the `LOCALSTACK_AUTH_TOKEN` environment variable in your `docker-compose.yml` file:
```

We should use the new version of docker compose. https://docs.docker.com/compose/#compose-v2-and-the-new-docker-compose-command

- If you want to continue using LocalStack without pro features you can set `ACTIVATE_PRO=0`.
```

> The key activation in LocalStack can fail for several reasons. The most common ones are documented below in this section.

Reviewer: ```suggestion
The key activation in LocalStack may fail for several reasons, and the most common ones are listed below in this section.
```

### Missing Credentials

You need to provide either an Auth Token or an API Key to start the LocalStack Pro image successfully. You can find your Auth Token or API Key on the [Auth Token page](https://app.localstack.cloud/workspace/auth-token) or the [Legacy API Key page](https://app.localstack.cloud/workspace/api-keys) in the LocalStack Web Application.

If you are using the `localstack` CLI, you can set the `LOCALSTACK_AUTH_TOKEN` environment variable to your Auth Token or use the following command to set it up:

{{< command >}}
$ localstack auth set-token <YOUR_AUTH_TOKEN>
{{< / command >}}

### Invalid License

> This issue occurs if there's no valid license linked to your account, possibly due to expiration or if it has not been assigned to you. You can check your license status on the [My License page](https://app.localstack.cloud/workspace/my-license) in the LocalStack Web Application.

Reviewer: ```suggestion
The issue may occur if there is no valid license linked to your account due to expiration or if the license has not been assigned. You can check your license status in the LocalStack Web Application on the [My License page](https://app.localstack.cloud/workspace/my-license).
```

### License Server Unreachable

> LocalStack attempts offline activation if the license server is inaccessible, but this necessitates re-activation every 24 hours. Log output might indicate that your machine cannot resolve the domain of the LocalStack API. You can this by using a tool like `dig`:

Reviewer: ```suggestion
LocalStack initiates offline activation when the license server is unreachable, requiring re-activation every 24 hours. Log output may indicate issues with your machine resolving the LocalStack API domain, which can be verified using a tool like `dig`:
```

{{< command >}}
$ dig api.localstack.cloud
{{< / command >}}

> If the result has some other status than `status: NOERROR,` your machine cannot resolve this domain. Some corporate DNS servers might filter requests to certain domains. Contact your network administrator to safelist `localstack.cloud` domains.

Reviewer: ```suggestion
If the result shows a status other than `status: NOERROR`, your machine is unable to resolve this domain. Certain corporate DNS servers may filter requests to specific domains. Kindly reach out to your network administrator to safelist `localstack.cloud` domain.
```
